### DISCUSSION TITLE
Credentials for more than one app

### DISCUSSION THREAD

**Discussion by alexanderadam** [Kunzisoft/KeePassDX#1567] [Q&A] [answered]
How is it stored when credentials are used for more app?
Is it still `AndroidApp` but with one line per app identifier?
I'm asking for editing properly entries in KeePassXC.

And a second tiny question: if I'd use fingerprint authentication — where would the data be stored? Is this handled by the phone or by KeePassDX?

**Comment by J-Jamet** [comment, marked as answer]
- New custom field : AndroidApp_[X] with [X] incremental Int (ex: AndroidApp_1)
- The database data will still be in the .kdbx file, but here a link is made between the biometric recognition and the master password. It is hashed and sent to the device's Keystore, where only the application can read it.
